index.html

```diff
@@ -170,7 +176,7 @@
     </p>
     <p>
       Scripting is an optional building block in WoT and it is typically used in gateways or browsers that are able to run a <a>WoT Runtime</a> and
-      <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager">script management</a>, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory">Thing Directory</a>.
+      <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager">script management</a>, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory">TD Directory</a>.
```

Comment:
More a question: Why this change.. discovery seems to speak still about "Thing Directory".
https://w3c.github.io/wot-discovery/#exploration-td-type-thingdirectory

Reply:
@relu91 please check... IIRC we discussed in the call that we use TD Directory instead of Thing Directory.
I am fine with either, personally I like Thing Directory more.

Reply:
In the terminology, "Thing Directory" has been renamed to TDD: https://w3c.github.io/wot-discovery/#exploration-directory I suppose that this change is partly motivated by the abbreviations (TD vs. TDD), as otherwise the directory could be confused with the Thing Description.

Reply:
You are right though, @danielpeintner, that the renaming in the Discovery spec does not seem to be that consistent...